pkg/plugins/policies/xds/cluster.go

```diff
@@ -41,7 +43,7 @@ type ClusterBuilder struct {
 }
 
 func NewClusterBuilder() *ClusterBuilder {
-	return &ClusterBuilder{}
+	return (&ClusterBuilder{}).WithWeight(1)
```

Comment:
Why is the default weight 1? is this the envoy default? should we put it in a const?

Reply:
I believe it's just a safe value. With weight equal 0 traffic won't flow.
I believe if you are mixing clusters, where some don't have weights and others do, it's a safe assumption, that the ones without explicit weight, will have weight equal 1.